lib/traject/config/geo_config.rb

```diff
@@ -314,6 +320,12 @@ module Constants
   accumulator << record.stanford_mods.pub_year_int if accumulator.empty?
 end
 
+to_field 'dc_source_sm', mods_xpath('string(//rdf:RDF/rdf:Description/fedora:isMemberOfCollection/@rdf:resource)') do |record, accumulator|
```

Comment:
How do you `xpath` again?

Reply:
Can we use this? https://github.com/sul-dlss/searchworks_traject_indexer/blob/master/lib/sdr_stuff.rb#L165-L169

Reply:
Meaning use that same xpath, or to extract that method into something that both indexers can use?

Reply:
I _think_ we already have access to that method, no?

Reply:
We use a related method like this: https://github.com/sul-dlss/searchworks_traject_indexer/blob/master/lib/traject/config/geo_config.rb#L156 ?

Reply:
Ah, I was miss-reading that file defined multiple classes and thought. that was tied to the SDR config.  We're not using that class anywhere, but I'll see how we've implemented it elsewhere.  👍

Reply:
And now I see that record is that thing.  It has been awhile since I have trajected, so took me a minute to remember how any of this works.